socorro/external/fs/filesystem.py

```diff
@@ -119,13 +121,14 @@ def defaultAcceptanceFunction(dummy):
     return True
 
 
+# FIXME(willkg): This is only used in the submtter. We should delete it or rewrite with os.walk.
```

Comment:
I'm pretty sure the submitter isn't set up to run in the local development environment. Otherwise I'd do this now.

There's another bug where we want to adjust the submitter so that it submits to multiple environments. I suspect it'll be easier to just rewrite the submitter using what I built for Antenna. If that happens, we can delete this rather than rewrite it.